src/vcpkg-test/dependencies.cpp

```diff
@@ -707,10 +705,10 @@ TEST_CASE ("version parse relaxed", "[versionplan]")
 TEST_CASE ("version parse date", "[versionplan]")
 {
     auto version_basic = Versions::DateVersion::from_string("2020-12-25");
-    check_date_version(version_basic, "2020-12-25", "", {});
+    check_date_version(version_basic, "2020-12-25", {});
 
     auto version_identifiers = Versions::DateVersion::from_string("2020-12-25.1.2.3");
```

Comment:
Hmmm by our current docs this shouldn't parse, it doesn't meet the date contract. (But resolving things in favor of what the code does rather than what the docs say is probably reasonable)

Reply:
I remember these were part of the initial spec draft and at some point even allowed for things like `2020-12-25.nightly` and `2020-12-25-nightly`,  but we decided on only allowing numeric identifiers after `.` to solve the sort behavior problem. If we decided to drop them entirely it was probably after the code had been implemented and I never got around to remove it.